File: dataset/output-bk/OPENAI_gpt-4o/Task125_OPENAI_gpt-4o.c
#include <stdio.h>
#include <string.h>

int main() {
    char userInput[50];
    char *resource1 = "resource1";
    char *resource2 = "resource2";

    printf("Enter your resource: ");
    scanf("%s", userInput);

    if (strcmp(userInput, resource1) == 0) {
        printf("Access to Resource 1
");
    } else if (strcmp(userInput, resource2) == 0) {
        printf("Access to Resource 2
");
    } else {
        printf("Resource not found
");
    }

    return 0;
}

File: dataset/output/OPENAI_gpt-4o/Task125_OPENAI_gpt-4o.java
import java.util.HashMap;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        HashMap<String, String> resources = new HashMap<>();
        resources.put("resource1", "Access to Resource 1");
        resources.put("resource2", "Access to Resource 2");

        System.out.print("Enter your resource: ");
        String userInput = scanner.nextLine();
        System.out.println(resources.getOrDefault(userInput, "Resource not found"));
    }
}